### PR TITLE
BZ 1283214 - After Force Unlock locked user can continue to work with Business Process but can't to save it

### PR DESCRIPTION
**Pull request by tsurdilo**
@csadilek PR for master

### CODE REVIEW

**Comment by manstis** [comment]
Does DesignerWidgetView contain a duplicate comparison in the lock require method? Sorry on my phone and cannot comment on the specific line.

File: jbpm-designer-client/src/main/java/org/jbpm/designer/client/DesignerViewImpl.java [hunk 1]
```diff
@@ -58,6 +58,9 @@ public void raiseEventSave() {
     }
 
     @Override
+    public void raiseEventReaquireLock() { designerWidget.raiseEventReaquireLock(designerWidget.getEditorID()); }
+
```
Comment (csadilek) [inline jbpm-designer-client/src/main/java/org/jbpm/designer/client/DesignerViewImpl.java]: There's code formatting issue here and acquire is misspelled ;).

**Comment by csadilek** [comment]
hi @manstis, not sure what you mean by the duplicate comparison?

**Comment by manstis** [comment]
Look at raiseEventReaquireLock I'm sure there's a duplicate check in the if statement.

**Comment by manstis** [comment]
$wnd.getElement.. sorry really difficult to paste/comment line on my phone :-(

**Comment by csadilek** [comment]
@manstis I don't see it.

**Comment by manstis** [comment]
No my bad. Oryx and Oryx.editor...I'll go back to sleep. Sorry Tiho for the spam.

**Comment by manstis** [comment]
Lesson learned.. don't review PRs on my phone.

**Comment by tsurdilo** [comment]
@csadilek @manstis amended with requested changes

**Comment by tsurdilo** [comment]
@csadilek added last formatting request

**Comment by csadilek** [comment]
Thanks, @tsurdilo. 

+1 ready to merge.

**Comment by csadilek** [comment]
Thanks.

master: b2e3187891a524bd8b827b8d96456a91640b68a7

**Pull request closed without merging**